### ISSUE TITLE
Changes in Medium 1.0

### ISSUE THREAD

**Issue by abitdodgy** [mduvall/grande.js#63]
Is Grande.js looking for parity with Medium in terms of features? If so, this means a couple of new features concerning images since Medium changed their interface.

Images can now be floated inline, floated inline and offset to the left, small, large, or cover mode, taking the full width of the screen. Attached is a screen shot.

![screen shot 2013-12-05 at 01 08 44](https://f.cloud.github.com/assets/545430/1679741/9c92abc4-5d5a-11e3-9108-2ad865b8fc66.png)

**Comment by mduvall** [comment]
Yep, going through 1.0 release right now and sorting out what they've added. The image alignment is definitely cool.
